bulletin_board/server/README.md

```diff
@@ -11,7 +11,118 @@ The Bulletin Board is a service composed by an Encryption Engine and an Append-O
 
 The Bulletin Board depends on the existence of a Decidim installation.
 
-- Installation:
+
+## Runining on Docker
+
+You can run this application conveniently packaged with docker compose.
+
+You can use the existing [docker-composer.yml](docker-composer.yml) as start point.
+It requires you to define a `.env` file with the following ENV variables:
+
+```
+SECRET_KEY_BASE=-your-rails-application-secret-
+DATABASE_URL=postgres://USERNAME:PASSWOR@DATABASE/BULLETIN_DB
+IDENTIFICATION_PRIVATE_KEY=-private-key-for-this-bulletin-board
+
+```
+
+#### Notes:
+
+- Generate `SECRET_KEY_BASE` with `rails secret`
+- Generate `IDENTIFICATION_PRIVATE_KEY` with `bin/rails client:generate_identification_private_key`
+
+### Configuration steps in this server (bulletin app):
+
+- Initialize the database, database is not included in this docker compose file but it should be pretty straithforward.
+  In order to initialize the database you need to run the comand `bin/rails db:create db:migrate`
+
+- In you Decidim server (the one you want to connect with this one), execute
+	`bin/rails decidim_elections:generate_identification_keys` (save the private key in you decidim environment)
+	copy the public key as is presented (url encoded) and, now in this server, execute:
+
+	`bin/rails 'client:add_authority[Authority name, public-key]'`
+
+	Note that "Authority name" can be anything as long is the same in Decidim than here.
+
+	**Important**: This step throws also an `API KEY` secret that needs to be configured in Decidim too.
+
+#### Running the app with docker-compose
+
+
+```bash
+docker-compose up -d
+```
+
+Note that you still need to configure some proxy around it.
+For instance a simple Nginx conf can be:
+
+```
+server {
+    server_name bulletin.audit.platoniq.net;
+    client_max_body_size 32M;
+    listen 80 ;
+    listen [::]:80 ;
+
+    #listen [::]:443 ssl ipv6only=on; # managed by Certbot
+    #listen 443 ssl; # managed by Certbot
+    #ssl_certificate ...
+    #ssl_certificate_key ...
+    #include /etc/letsencrypt/options-ssl-nginx.conf; # managed by Certbot
+    #ssl_dhparam /etc/letsencrypt/ssl-dhparams.pem; # managed by Certbot
+
+    location / {
+        proxy_pass http://127.0.0.1:8000;
+   }
+}
+
+```
+
+Note: you should add SSL configuration, for instance with [letsencrypt](https://certbot.eff.org/lets-encrypt)
+
+This server is ready, you can access the hompage either at https://localhost:8000 or https:/you-server.com depending on your final configuration.
```

Comment:
```suggestion
This server is ready, you can access the hompage either at http://bulletin-board.example.org or https://bulletin-board.example.org depending on your final configuration.
```

Reply:
Actually, the localhost is relevant here, more like to change it to http://localhost:8000 or http(s)//bulletin-board.example.org